### PR TITLE
fix: and enable dynamic booking test

### PR DESCRIPTION
**Pull request by alannnc**
## What does this PR do?

- Enabling dynamic booking test to run again with new booker.
- Fix getBookingForReschedule()

Fixes # (issue)
## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/features/bookings/lib/get-booking.ts [hunk 1]
```diff
@@ -109,63 +109,20 @@ export const getBookingWithResponses = <
...
-          uid,
-        },
-        select: {
-          eventTypeId: true,
-        },
-      })
-    )?.eventTypeId || null;
-
-  // If no booking is found via the uid, it's probably a booking seat,
-  // which we query next.
```
Comment (alannnc) [inline packages/features/bookings/lib/get-booking.ts:127]: Shouldn't query seats here anymore.

File: packages/features/bookings/lib/get-booking.ts [hunk 2]
```diff
@@ -109,63 +109,20 @@ export const getBookingWithResponses = <
 export default getBooking;
 
 export const getBookingForReschedule = async (uid: string) => {
-  let eventTypeId: number | null = null;
-  let rescheduleUid: string | null = null;
```
Comment (alannnc) [inline packages/features/bookings/lib/get-booking.ts:113]: rescheduleUid no longer needed.

**Review by emrysal — DISMISSED** [review]
Nice 👍

File: packages/features/bookings/lib/get-booking.ts [hunk 3]
```diff
@@ -109,59 +109,20 @@ export const getBookingWithResponses = <
...
-          },
-        },
-      },
-    });
-    if (bookingSeat) {
-      rescheduleUid = bookingSeat.booking.uid;
-      attendeeEmail = bookingSeat.attendee.email;
-    }
-  }
+  if (!bookingFound) return null;
```
Comment (hariombalhara) [inline packages/features/bookings/lib/get-booking.ts:121]: `getBooking` does it already. I think we can just delete this code.

**Review by hariombalhara — DISMISSED** [review]
Left a comment. Also, reschedule tests are failing, it might be related.

**Comment by alannnc** [comment]
> Left a comment. Also, reschedule tests are failing, it might be related.

@hariombalhara Undo some changes as seem they were necesary. Tests should be fine now.

**Review by zomars — APPROVED** [review]

**Pull request merged**